tasks/Module_2/task_1.py

```diff
@@ -0,0 +1,17 @@
+stuff = {'rope': 1, 'torch': 6, 'gold coin': 42, 'dagger': 1, 'arrow': 12}
+def display_inventory(inventory):
+    print("Inventory:")
```

Comment:
I'd move this print strictly above the for loop on line 6. It's good to keep a similar logic together.